pkg/proto/configuration/grpc/grpc.proto

```diff
@@ -51,6 +55,30 @@ message ClientKeepaliveConfiguration {
   bool permit_without_stream = 3;
 }
 
+message ClientOAuthConfiguration {
+  // Use default google credentials (false by default)
+  // See https://developers.google.com/accounts/docs/application-default-credentials
+  // Either google_default_credentials should be set to True
+  // or json_key or json_key_file should be provided.
+  bool google_default_credentials = 1;
+
+  // Service account private key to use
+  // to obtain access token.
+  // Either google_default_credentials should be set to True
+  // or json_key or json_key_file should be provided.
+  string json_key = 2;
+
+  // File with service account private key to use
+  // to obtain access token.
+  // Either google_default_credentials should be set to True
+  // or json_key or json_key_file should be provided.
+  string json_key_file = 3;
```

Comment:
This option is superfluous, right? It's also possible to use `jsonKey: importstr 'filename'` from within Jsonnet.

Reply:
@EdSchouten you're correct, but it will work for jsonnet only. What if you'll switch from jsonnet to pure json or yaml after some time?

Reply:
it's ok to remove this option for me, though :)

Reply:
Then this won't be the only piece of code that needs adjusting. Notice how the TLS client and server also use Jsonnet `importstr` to load key material.

Reply:
Also to answer your question in more detail: it's unlikely that we ever get rid of Jsonnet and not replace it with something that's on par with it. Still, assuming we would get rid of Jsonnet (ad absurdum), then I'd rather see that we work towards a universal solution for loading key material from disk. For example, by adding a:

```proto
message KeyMaterial {
  oneof source {
    string direct = 1;
    string from_disk = 2;
  }
}
```

Reply:
option removed